packages/algob/src/lib/tx.ts

```diff
@@ -213,6 +213,41 @@ export async function transferMicroAlgosLsig (
   return await deployer.waitForConfirmation(pendingTx.txId);
 }
 
+/**
```

Comment:
no need for this function, you can use `transferMicroAlgosLsig` already implemented in tx

Reply:
Please remove it

Reply:
you can change `fromAccount: AccountSDK,` to `fromAccount: string`

Reply:
Alright